src/dev/run_i18n_check.js

```diff
@@ -17,13 +17,45 @@
  * under the License.
  */
 
-import { run } from './run';
-import { extractDefaultTranslations } from './i18n/extract_default_translations';
+import chalk from 'chalk';
+import Listr from 'listr';
+import { resolve } from 'path';
+
+import { run, createFailError } from './run';
+import {
```

Comment:
__nit:__ it probably makes sense to have `./i18n/index.js` that re-exports everything we may need here so that we can combine all these imports into one.